pkg/cri/io/logger.go

```diff
@@ -112,17 +117,24 @@ func readLine(b *bufio.Reader) (line []byte, isPrefix bool, err error) {
 	return
 }
 
-func redirectLogs(path string, rc io.ReadCloser, w io.Writer, s StreamType, maxLen int) {
+func redirectLogs(path string, rc io.ReadCloser, w io.Writer, s StreamType, maxLen int, logLimit int, logBurst int) {
 	defer rc.Close()
 	var (
-		stream    = []byte(s)
-		delimiter = []byte{delimiter}
-		partial   = []byte(runtime.LogTagPartial)
-		full      = []byte(runtime.LogTagFull)
-		buf       [][]byte
-		length    int
-		bufSize   = defaultBufSize
+		stream      = []byte(s)
+		delimiter   = []byte{delimiter}
+		partial     = []byte(runtime.LogTagPartial)
+		full        = []byte(runtime.LogTagFull)
+		buf         [][]byte
+		length      int
+		bufSize     = defaultBufSize
+		rateLimiter *rate.Limiter
+		limitedNum  int
+		writeLog    func(tag, line []byte)
 	)
+	if logBurst > 0 && logLimit > 0 {
```

Comment:
Is it an error if one is set and the other is not?

Reply:
Sorry for late reply. We could set default value in `criconfig.DefaultConfig()`, like
`ContainerLogLimit = 0`
`ContainerLogBurst = 0`
which means there is no limitation on the speed of writing container logs to disk.

Reply:
Updated (set default value in `criconfig.DefaultConfig()`)